requirements.txt

```diff
@@ -1,4 +1,4 @@
--e git+https://github.com/tejado/pgoapi.git@v1.1.0#egg=pgoapi
+-e git+https://github.com/tejado/pgoapi.git@81e786cabf027a1c8fbd1e9a07e1c11aa3d8ee8b#egg=pgoapi
```

Comment:
I need a commit later than 1.1.0 for these features to work, Locking it to a specific commit makes sure things won't break in the future.